tensorflow/python/ops/signal/fft_ops.py

```diff
@@ -324,6 +325,83 @@ def _grad(op, grad):
 
   return _grad
 
+@tf_export("signal.fftshift")
+def fftshift(x, axes=None):
```

Comment:
TF ops should take a name=None argument so users can choose the op name in the graph / tensorboard